.env.example

```
@@ -18,20 +55,23 @@ REDIS_HOST=localhost
 REDIS_PORT=6379
 
 # Microsoft Graph Configuration
-MSGRAPH_CLIENT_ID=your-client-id-here
-MSGRAPH_TENANT_ID=your-tenant-id-here
-MSGRAPH_CLIENT_SECRET=your-client-secret-here
+# SECURITY CRITICAL: Get these from Azure App Registration
+# Portal: https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps
+MSGRAPH_CLIENT_ID=your-azure-client-id-here
+MSGRAPH_TENANT_ID=your-azure-tenant-id-here
+MSGRAPH_CLIENT_SECRET=generate-using-scripts/generate-secure-secrets.js
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MSGRAPH_CLIENT_SECRET key should go before the MSGRAPH_TENANT_ID key

(UnorderedKey)